### DISCUSSION TITLE
Passkey support

### DISCUSSION THREAD

**Discussion by ranok** [bookwyrm-social/bookwyrm#3015] [Ideas]
Passkeys are slowly replacing passwords as a more smooth and secure option. It would be great to let users enroll, then use passkeys on their devices to authenticate to Bookwyrm.

**Comment by roke-julian-lockhart** [comment]
> https://github.com/bookwyrm-social/bookwyrm/discussions/3015#discussion-5691985

Wouldn't creating a https://github.com/bookwyrm-social/bookwyrm/issues/new?assignees=&labels=&projects=&template=feature_request.md&title= for this make more sense, @ranok? https://github.com/bookwyrm-social/bookwyrm/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+passkey returns 0 results.
